pilot/pkg/networking/plugin/authz/rbac.go

```diff
@@ -80,8 +80,11 @@ const (
 	attrDestUser      = "destination.user"      // service account, e.g. "bookinfo-productpage".
 	attrConnSNI       = "connection.sni"        // server name indication, e.g. "www.example.com".
 
+	// attributes that could be used in a ServiceRole rules.
 	methodHeader = ":method"
 	pathHeader   = ":path"
+	hostHeader   = ":authority"
+	port         = "port"
```

Comment:
port => portKey? Just to be clear it is a constant.

Reply:
Done!